docs/upgrading_to_v1.0.md

```diff
@@ -0,0 +1,8 @@
+# Upgrading to v1.0.0
```

Comment:
For pre 1.0 major releases, you can continue to use 0.x tags. If you think you are ready to tag 1.0 feel free to go ahead but thought I would call out that it is not a requirement.

Reply:
With module covering almost all the available features and used by many customers we can move it to 1.0.